lib/daru/vector.rb

```diff
@@ -762,25 +762,25 @@ def replace_nils! replacement
 
     # Lags the series by k periods.
     #
-    # The convention is to set the oldest observations (the first ones
-    # in the series) to nil so that the size of the lagged series is the
-    # same as the original.
+    # Lags the series by k periods where k can be a positive or negative
+    # integer.
+    # Returns a new series with nil values inserted into the series
+    # keeping the size of the new series the same as the original.
```

Comment:
Let's update the docs a bit. Probably this way (minus my broken English):
```ruby
# Lags the series by `k` periods, "shifting" data and inserting `nil`s 
# from beginning or end of a vector, while preserving original vector's size.
#
# `k` could be positive or negative number. If `k` is positive, `nil`s are inserted 
# at the beginning of the vector, otherwise they are inserted at the end.
```